clientv3/integration/kv_test.go

```diff
@@ -28,6 +29,53 @@ import (
 	"golang.org/x/net/context"
 )
 
+func assertRpctypesError(t *testing.T, err, expected error) {
+	if err == nil {
```

Comment:
this line is unnecessary. what if expected is also nil? if expected is not nil, then line 36 will do the job.